spec/dry_spec.rb

```diff
@@ -0,0 +1,37 @@
+CONSTRUCTOR_TYPES = %i[permissive schema strict strict_with_defaults].freeze
```

Comment:
No need to define a top level const like this. It'd be better to just put it in a `let(:constructor_types)`.

Reply:
That wouldn't quite work for the way I used it, I just inlined it.